### ISSUE TITLE
Can't detect images sometime

### ISSUE THREAD

**Issue by Haow235** [jsbroks/coco-annotator#321]
My problem is that it sometimes can't find the image.
OS: Linux
what I do:
  1. start server;
  2. create a dataset named xxx
  3. cp image files to the directory ./dataset/xxx/
sometimes, it can detect the image, sometimes it just show "No image in dataset"

**Comment by rbavery** [comment]
Hi @jsbroks I'm also running into this issue when testing the most recent production build with tif imagery.

Tasks are stuck running trying to discover but they don't find anything

![Selection_003](https://user-images.githubusercontent.com/22258697/73023961-6cc4e380-3de1-11ea-8b72-b9acf61f7d5c.png)

![Selection_002](https://user-images.githubusercontent.com/22258697/73023974-70f10100-3de1-11ea-8f4d-c54cee00700a.png)

But in `datasets/planet-4band-test` I have both a 1 channel and 4 channel image. 

Metadata for 4 channel image

```
coco-annotator/datasets/planet-4band-test  on  master via 🅒 base 
➜ rio info 20180801_173117_1009_3B_AnalyticMS_SR_clip.tif
{"blockxsize": 256, "blockysize": 256, "bounds": [570219.0, 3496098.0, 585822.0, 3500655.0], "colorinterp": ["blue", "green", "red", "undefined"], "compress": "lzw", "count": 4, "crs": "EPSG:32612", "descriptions": [null, null, null, null], "driver": "GTiff", "dtype": "uint16", "height": 1519, "indexes": [1, 2, 3, 4], "interleave": "pixel", "lnglat": [-110.1774019529404, 31.617889203049], "mask_flags": [["nodata"], ["nodata"], ["nodata"], ["nodata"]], "nodata": 0.0, "res": [3.0, 3.0], "shape": [1519, 5201], "tiled": true, "transform": [3.0, 0.0, 570219.0, 0.0, -3.0, 3500655.0, 0.0, 0.0, 1.0], "units": [null, null, null, null], "width": 5201}
```

I'm also unable to detect a .tif image that has 3 channels, so I think this is a general tif issue, though I saw recently that it looked like there was a PR to add tif support.
This issue can be tested using publicly available tif data from this link: https://ndownloader.figshare.com/articles/2009586/versions/10

RGB tif imagery is stored at `~/geospatial-python/data/NEON-DS-Airborne-Remote-Sensing/HARV/RGB_Imagery` from the data available via the above link.

As a side note, it'd be nice if COCO annotator was channel agnostic and could handle 4 channel imagery similar to [napari](https://github.com/napari/napari) (an ND image viewer), where different channel combinations could be selected for visualization, either on import or interactively during the labeling stage.

**Comment by EtienneDavid** [comment]
@Haow235 did you try to use the "scan" button in the dataset view ?

**Comment by Haow235** [comment]
@EtienneDavid Yes, I tried "Scan", "refresh", "import", and they don't work too. I tried update then restart my computer, restart the server, and recreate the dataset, then it can detect the images.

**Comment by rbavery** [comment]
Scanning and restarting does not work for me when using .tif images.

**Comment by EtienneDavid** [comment]
OK, it may linked to a docker volume management problem...

**Comment by Mantezzy** [comment]
I also ran into the same issue. I only have 1 image in the folder and it can't detect it.

**Comment by wish2023** [comment]
I have the same issue too

**Comment by aleksod** [comment]
Same :(

**Comment by manar-majdi** [comment]
help i use os: Windows 11 and it 's can't  find all my image :(
